### ISSUE TITLE
amp-story: Missing or inaccurate transcript

### ISSUE THREAD

**Issue by TetraLogicalHelpdesk** [ampproject/amphtml#32495]
# Missing or inaccurate transcript

## WCAG Level
Level A

## Priority
Low

## Pages/screens/components affected
* [Audio - amp-story background audio](https://www.gstatic.com/amphtml/stamp/qa/audio-story.html)
* [Audio - amp-story-page background audio](https://www.gstatic.com/amphtml/stamp/qa/audio-page.html)
* [Audio - amp-story using amp-audio](https://www.gstatic.com/amphtml/stamp/qa/amp-audio.html)

## Description
When a transcript is not provided for audio-only content, like a podcast, song, or a recording of a radio broadcast, people who cannot hear the audio will find it difficult or impossible to understand.

A transcript is a text alternative to the audio content. Typically it includes the words spoken or sung in the audio content, plus descriptions of other important sounds. It may also contain a general description of the audio content - the genre of music or the style of the presenter for example.

Three of the tested samples use meaningful audio content, but don't provide a text alternative/transcript for it. This issue is marked as "Low" priority, but only because of the sample nature of the audio-only content used.

## User impact
Without an accurate transcript that contains the words being spoken or sung, and descriptions of other important things like sound-effects, someone who is deaf or hard or hearing will not be able to understand the audio content.

Transcripts may also be useful to people in noisy environments where the ambient noise makes it hard to hear audio content, and to people in quiet environments where turning up the volume may disturb other people nearby.

## Required solution
Provide a text transcript for all audio-only content, and make sure that the dialogue and other important sounds are accurately represented.

This solution must be applied to all instances of the issue identified within the test sample, then applied to all other instances of the same issue identified throughout the rest of the website.

## Implementation guidance
This may be something that can be left up to authors themselves - providing a text version/transcript of the audio used in a story/page directly in the visible content. This may not be suitable though for long audio/dense content. A more systemic approach would be to provide an extra general control (similar to the sound on/off and play/pause control) that allows users to explicitly view a transcript of audio used (and giving authors a way of defining this content when creating AMP stories).

## Test procedure(s)
Use these steps to confirm that the solution has been correctly applied to issues identified within the test sample, and to test the rest of the website for instances of the same issue:
1. Check that a text transcript is available for all audio-only content that contains words or other important sounds.
2. Check that the transcript is an accurate representation of the words and that other important sounds are accurately described.

### Definition of done
Complete all of these tasks before closing this issue or indicating it is ready for retest:
* All issues identified within the test sample have been resolved.
* The rest of the website has been tested for the same issue.
* All issues identified throughout the rest of the website have been resolved or filed as new issues.

## Related standards
* [WCAG 2.1 Success Criterion 1.2.1 Audio-only and Video-only (Prerecorded) (Level A)]

## More information
* [Nomensa: What are transcripts?](https://www.nomensa.com/blog/2010/what-are-transcripts/)

## Test data
Test date: January 2021

**Comment by gmajoulet** [comment]
This is unfortunately publisher provided content. Our accessibility guidance recommends using captions and this is a supported feature, but I don't believe there is anything more we can do at this point, except automatic caption/transcript generation, but this goes beyond this ticket.
